### PR TITLE
Lstm baseline changes

### PR DESCRIPTION
**Pull request by erenkiliclar**
Plese have a look into the additions & amendments. Thanks

### CODE REVIEW

File: Dockerfile [hunk 1]
```diff
@@ -0,0 +1,39 @@
...
+#      Tensorflow version
+# FROM tensorflow/tensorflow:2.10.0
+
+#      Or tensorflow to run on Apple Silicon (M1 / M2)
+# FROM armswdev/tensorflow-arm-neoverse:r22.11-tf-2.10.0-eigen
+
+
+# Copy everything we need into the image
+COPY baseline_model baseline_model
+COPY api api
```
Comment (michafdlr) [inline Dockerfile]: api folder is not yet created, right?

**Review by michafdlr — COMMENTED** [review]
Looks good. We will also have to update the requirements (uvicorn and probably later FastAPI and also the Cloud packages), right?

**Comment by erenkiliclar** [comment]
> Looks good. We will also have to update the requirements (uvicorn and probably later FastAPI and also the Cloud packages), right?

exactly. I've created the Cloud project and its bucket

File: Dockerfile (hunk 1 above)
Comment (erenkiliclar) [reply Dockerfile]: api folder created containing a fast.py file.

File: Dockerfile [hunk 2]
```diff
@@ -0,0 +1,39 @@
+# TODO: select a base image
+# Tip: start with a full base image, and then see if you can optimize with
+#      a slim or tensorflow base
+
+#      Standard version
+FROM python:3.10.10-buster
```
Comment (michafdlr) [inline Dockerfile:6]: Will it be a problem that we are running Python version 3.10.6 in our venv?

Comment (erenkiliclar) [reply Dockerfile:6]: It shouldn't cause any issues as 3.10.10 is used as a based image for our application container's Python version. Also minor version differences do not introduce significant changes that impact the compatibility.

Comment (michafdlr) [reply Dockerfile:6]: Cool, was not sure about it

Comment (erenkiliclar) [reply Dockerfile:6]: that was a good ask.

**Pull request merged**